.github/workflows/check-full.yaml

```diff
@@ -20,9 +20,8 @@ jobs:
         config:
           - {os: macos-latest,   r: 'release'}
 
+          - {os: windows-latest, r: 'devel'}
```

Comment:
I think we should always be testing on windows-devel.

It's currently failing because of this issue: https://github.com/gagolews/stringi/issues/508